kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/KafkaResponseUtils.java

```diff
@@ -184,22 +188,52 @@ public static DescribeGroupsResponse newDescribeGroups(
         return new DescribeGroupsResponse(data);
     }
 
-    public static FindCoordinatorResponse newFindCoordinator(Node node) {
+    public static FindCoordinatorResponse newFindCoordinator(List<String> coordinatorKeys,
```

Comment:
Can we reuse this method? It looks like this method is not used anymore.

Reply:
I create a new method for generating `Coordinator` results.

Reply:
if the method is not used then we can drop it.